x-pack/solutions/security/plugins/security_solution/common/api/endpoint/actions/response_actions/run_script/run_script.ts

```diff
@@ -54,5 +61,42 @@ export const RunScriptActionRequestSchema = {
     ),
   }),
 };
+export const MSDefenderEndpointRunScriptActionRequestSchema = {
```

Comment:
Remember to add tests for schema validations in `x-pack/solutions/security/plugins/security_solution/common/endpoint/schema/actions.test.ts`